files-community/Files
# Feature: Added support for FSCTL_SET_COMPRESSION

### Resolved / Related Issues

- Closes #15976

### Steps used to test these changes

1. Open Files app
2. Open a file from NTFS filesystem
3. Set/unset the value
4. Open a file from other filesystems
5. See that option is disabled
